eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -0,0 +1,145 @@
+<#
+.SYNOPSIS
+Updates package README.md for publishing to docs.microsoft.com
+
+.DESCRIPTION
+Given a PackageInfo .json file, format the package README.md file with metadata
+and other information needed to release reference docs: 
+
+* Adjust README.md content to include metadata
+* Insert the package verison number in the README.md title 
+* Copy file to the appropriate location in the documentation repository
+* Copy PackageInfo .json file to the metadata location in the reference docs
+  repository. This enables the Docs CI build to onboard packages which have not
+  shipped and for which there are no entries in the metadata CSV files.
+
+.PARAMETER ArtifactLocation
+Location of the artifact information .json file. This is usually stored in build
+artifacts under packages/PackageInfo/<package-name>.json.
+
+.PARAMETER DocRepoLocation 
+Location of the root of the docs.microsoft.com reference doc location. Further
+path information is provided by $GetDocsMsMetadataForPackageFn
+
+.PARAMETER Language
+Programming language to supply to metadata
+
+.PARAMETER RepoId
+GitHub repository ID of the SDK. Typically of the form: 'Azure/azure-sdk-for-js'
+
+#>
+
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$ArtifactLocation,
+  
+  [Parameter(Mandatory = $true)]
+  [string]$DocRepoLocation, 
+
+  [Parameter(Mandatory = $true)]
+  [string]$Language,
+
+  [Parameter(Mandatory = $true)]
+  [string]$RepoId
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+
+$releaseReplaceRegex = "(https://github.com/$RepoId/(?:blob|tree)/)(?:master|main)"
+$TITLE_REGEX = "(\#\s+(?<filetitle>Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C)))"
+
+function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata) {
+  # Normalize service name "Key Vault" -> "keyvault"
+  # TODO: Use taxonomy for service name -- https://github.com/Azure/azure-sdk-tools/issues/1442
+  # probably from metadata
+  $service = $PackageMetadata.ServiceName.ToLower().Replace(" ", "")
+  # Generate the release tag for use in link substitution
+  $tag = "$($PackageInfo.Name)_$($PackageInfo.Version)"
```

Comment:
This is a bad assumption because not all tags are formatted as such. Once this goes into common we should see about getting this data from the package-properties instead of assuming this format.

Reply:
Created issue https://github.com/Azure/azure-sdk-tools/issues/1727 to track